src/nile/common.py

```diff
@@ -38,3 +38,10 @@ def get_all_contracts(ext=None):
         ]
 
     return files
+
+
+def logger(verbose):
```

Comment:
Should we use the Python's logger api? I think for now it's overkill.

Reply:
heh I thought i was reinventing the wheel. you mean [this one](https://docs.python.org/3/library/logging.html) right?

Reply:
removed this alltogether. might add it back later